src/page/Auth/Signup/view/OwnerDataPage/index.tsx

```diff
@@ -0,0 +1,51 @@
+import useMediaQuery from 'utils/hooks/useMediaQuery';
+import styles from './OwnerData.module.scss';
+
+export default function OwnerData() {
+  const { isMobile } = useMediaQuery();
+  return (
+    <section className={styles.form}>
+      <div>
+        <span className={styles.form__label}>대표자명</span>
+        <input className={styles.form__input} type="text" placeholder={isMobile ? '대표자명(실명)' : ''} />
+      </div>
+      <div>
+        <span className={styles.form__label}>사업자등록번호</span>
+        {!isMobile ? (
+          <div className={styles['input-wrapper']}>
+            <div className={`${styles['form__input__registration-number']}`}>
+              <input className={`${styles['form__input__registration-number--first']}`} type="text" maxLength={3} />
```

Comment:
```suggestion
            <div className={styles['form__input__registration-number']}>
              <input className={styles['form__input__registration-number--first']} type="text" maxLength={3} />
```
classname을 변수로 넣으신 이유가 있을까요?

Reply:
잘못 사용한 부분인 것 같습니다. 수정 완료했습니다. b27a4d3e56b167aeb6db17576faf03c644329f19